QADI_README.md

```
@@ -0,0 +1,63 @@
+# QADI Command Line Tool
+
+A command-line tool that uses Google's Gemini API to analyze questions using the QADI (Question, Abduction, Deduction, Induction) methodology.
+
+## Setup
+
+1. Add your Google API key to `.env`:
+```bash
+GOOGLE_API_KEY=your-google-api-key-here
+```
+
+2. Ensure dependencies are installed:
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)